source/uwp/Renderer/lib/MediaHelpers.cpp

```diff
@@ -49,45 +66,154 @@ void GetMediaPosterAsImage(
     THROW_IF_FAILED(posterAsImage.CopyTo(posterImage));
 }
 
-void AddPlayButtonIcon(
-    IPanel * mediaPanel)
+void AddDefaultPlayIcon(
+    IPanel * posterPanel,
+    IAdaptiveHostConfig * hostConfig,
+    IAdaptiveRenderArgs* renderArgs)
 {
+    // Create a rectangle
+    ComPtr<IRectangle> rectangle = XamlHelpers::CreateXamlClass<IRectangle>(HStringReference(RuntimeClass_Windows_UI_Xaml_Shapes_Rectangle));
+
+    // Set the size
+    ComPtr<IFrameworkElement> rectangleAsFrameworkElement;
+    THROW_IF_FAILED(rectangle.As(&rectangleAsFrameworkElement));
+    THROW_IF_FAILED(rectangleAsFrameworkElement->put_Height(c_playIconSize));
+    THROW_IF_FAILED(rectangleAsFrameworkElement->put_Width(c_playIconSize));
+
+    // Round the corners
+    THROW_IF_FAILED(rectangle->put_RadiusX(c_playIconCornerRadius));
+    THROW_IF_FAILED(rectangle->put_RadiusY(c_playIconCornerRadius));
+
```

Comment:
I like the icon, but let's have a validation that this is how we want to ship.

Reply:
I'll follow up with Matt.

---
In reply to: [192467417](https://github.com/Microsoft/AdaptiveCards/pull/1509#discussion_r192467417) [](ancestors = 192467417)